test/src/test/java/jenkins/security/Security218CliTest.java

```diff
@@ -133,11 +134,13 @@ public void probeJRMPListener() throws Exception {
         probe(Payload.JRMPListener, 3);
     }
 
+    //TODO: disabled since Corba is removed from Java 11
     @PresetData(PresetData.DataSet.ANONYMOUS_READONLY)
     @Test
     @Issue("SECURITY-317")
+    @Ignore
```

Comment:
Why not delete the method since CORBA is removed from Java 11?

Reply:
Because we still need to merge it back to upstream. I am not sure how we approach this particular case